src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -55,6 +62,85 @@ const ApplicationLauncher = _.flow(
+        }
+        currentMetadata[hashedOwnerEmail] = 'doNotSync'
+        await Ajax().Buckets.analysis(googleProject, bucketName, stripExtension(file), tools.RStudio.label).updateMetadata(file, currentMetadata)
+      }, outdatedAnalyses))
+      onDismiss()
+    })
+
+    const getDisplayList = analyses => {
+      let res = ''
+      let i = 1
+      const end = analyses?.length
+      for (const analysis of analyses) {
+        res += analysis?.name.split('/')[1]
+        if (i < end) {
+          res += ', '
+        }
+        i++
+      }
+      return res
+    }
+
+    return h(Modal, {
+      onDismiss,
+      width: 530,
+      title: outdatedAnalyses?.length > 1 ? 'R Markdown Files In Use' : `R Markdown File Is In Use`,
+      showButtons: false
+    }, [
+      Utils.cond(
+        [outdatedAnalyses?.length > 1, () => [p(`These R markdown files are being edited by another user and your versions are now outdated. Your files will no longer sync with the workspace bucket.`),
+          p(getDisplayList(outdatedAnalyses)),
+          p('You can'),
+          p(['1) ', b('save your changes as new copies'), ' of your files which will enable file syncing on the copies']),
+          p([b('or')]),
+          p(['2) ', b('continue working on your versions'), ` of ${getDisplayList(outdatedAnalyses)} with file syncing disabled.`])]],
+        [outdatedAnalyses?.length === 1, () => [p(`${outdatedAnalyses[0].name.split('/')[1]} is being edited by another user and your version is now outdated. Your file will no longer sync with the workspace bucket.`),
+          p('You can'),
+          p(['1) ', b('save your changes as a new copy'), ` of ${outdatedAnalyses[0].name.split('/')[1]} which will enable file syncing on the copy`]),
+          p([b('or')]),
+          p(['2) ', b('continue working on your outdated version'), ` of ${outdatedAnalyses[0].name.split('/')[1]} with file syncing disabled.`])]]),
+      div({ style: { marginTop: '2rem' } }, [
+        h(ButtonSecondary, {
+          style: { padding: '0 1rem' },
+          onClick: () => handleChoice(false)
+        }, ['Keep outdated version']),
+        h(ButtonPrimary, {
+          style: { padding: '0 1rem' },
+          onClick: () => handleChoice(true)
+        }, ['Make a copy'])
+      ])
+    ])
+  }
+
+  const checkForOutdatedAnalyses = async ({ googleProject, bucketName }) => {
+    const analyses = await Ajax(signal).Buckets.listAnalyses(googleProject, bucketName)
+    const rAnalyses = _.filter(({ name }) => name.endsWith(`.${tools.RStudio.ext}`), analyses)
+    return _.filter(
+      analysis => analysis && Object.keys(analysis).includes('metadata') && Object.keys(analysis.metadata).includes(hashedOwnerEmail) &&
+        analysis.metadata[hashedOwnerEmail] === 'outdated', rAnalyses)
```

Comment:
Would the following be simpler?
```suggestion
    return _.filter(analysis =>
      _.endsWith(`.${tools.RStudio.ext}`, analysis?.name) &&
      analysis?.metadata?.hashedOwnerEmail === 'outdated', analyses)
```